build/gitkubed/Dockerfile

```diff
@@ -4,10 +4,10 @@ MAINTAINER tiru@hasura.io
 # Install openssh server
 RUN apt-get update \
  && apt-get install -y upx-ucl binutils curl openssh-server git jq \
- && curl -o /tmp/docker-1.12 'https://get.docker.com/builds/Linux/x86_64/docker-1.12.6.tgz' \
- && tar -xf /tmp/docker-1.12 -C /tmp \
+ && curl -o /tmp/docker-17.05 'https://get.docker.com/builds/Linux/x86_64/docker-17.05.0-ce.tgz' \
+ && tar -xf /tmp/docker-17.05 -C /tmp \
  && mv /tmp/docker/docker /bin/docker \
- && rm -rf /tmp/docker-1.12 /tmp/docker \
+ && rm -rf /tmp/docker-17.05 /tmp/docker \
```

Comment:
Note that this can be simplified by using `COPY --from`, and copying the binary from the official Docker images.

For example; the following example pulls the `docker:18.06.1` image (if it's not available locally), copies the `docker` to `/bin/docker` in a `scratch` image;

```bash
docker build -t docker-client -<<EOF
FROM scratch
COPY --from=docker:18.06.1 /usr/local/bin/docker /bin/docker
ENTRYPOINT ["/bin/docker"]
EOF
```

Which, when bind-mounting `/var/run` (to give access to the local daemon's socket), gives a functional `docker` cli;


```
$ docker run --rm -v /var/run/:/var/run/ docker-client version
Client:
 Version:           18.06.0-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        0ffa825
 Built:             Wed Jul 18 19:04:39 2018
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:29:02 2018
  OS/Arch:          linux/amd64
  Experimental:     true
```